airootfs/etc/passwd

```diff
@@ -1,2 +1 @@
 root:x:0:0:root:/root:/usr/bin/zsh
-liveuser:x:1000:1000::/home/liveuser:/usr/bin/zsh
```

Comment:
Why is the liveuser removed? That's the user that starts Xorg

Reply:
Liveuser is unable to start sway with wayland due to lack of permissions so it was removed